code/modules/events/spacevine.dm

```diff
@@ -294,8 +305,117 @@
 		return
 	if(ishuman(crosser))
 		var/mob/living/carbon/human/H = crosser
-		H.slip(10)
+		H.slip(20)
 		to_chat(H, "<span class='alert'>The vines slip you!</span>")
+
+/datum/spacevine_mutation/teleporting
+	name = "teleporting"
+	hue = "#1105b6"
+	severity = 3
+	quality = NEGATIVE
+
+/datum/spacevine_mutation/teleporting/on_hit(obj/structure/spacevine/holder, mob/hitter, obj/item/I, expected_damage)
+	if(isliving(hitter))
+		var/mob/living/M = hitter
+		if(isvineimmune(M))
+			return
+		if(prob(25))
+			do_teleport(M, get_turf(M), 8, asoundin = 'sound/effects/phasein.ogg', channel = TELEPORT_CHANNEL_BLUESPACE)
+	. = expected_damage
+
+/datum/spacevine_mutation/meleereflecting
+	name = "melee reflecting"
+	hue = "#b6054f"
+	severity = 2
+	quality = NEGATIVE
+
+/datum/spacevine_mutation/meleereflecting/on_hit(obj/structure/spacevine/holder, mob/hitter, obj/item/I, expected_damage)
+	if(isliving(hitter))
+		var/mob/living/M = hitter
+		if(isvineimmune(M))
+			return
+		if(prob(10))
+			M.adjustBruteLoss(expected_damage)
+		else
+			. = expected_damage
+
+/datum/spacevine_mutation/seeding
+	name = "seeding"
+	hue = "#68b95d"
+	severity = 3
+	quality = NEGATIVE
+	
+/datum/spacevine_mutation/seeding/on_cross(obj/structure/spacevine/holder, mob/crosser)
+	if(isliving(crosser))
+		var/mob/living/M = crosser
+		if(isvineimmune(M))
+			return
+		if(prob(10))
+			addtimer(CALLBACK(M, /mob/living/proc/plant_kudzu), 1 MINUTES)
+
+/datum/spacevine_mutation/seeding/on_hit(obj/structure/spacevine/holder, mob/hitter, obj/item/I, expected_damage)
+	if(isliving(hitter))
+		var/mob/living/M = hitter
+		if(isvineimmune(M))
+			return
+		if(prob(10))
+			addtimer(CALLBACK(M, /mob/living/proc/plant_kudzu), 1 MINUTES)
+	. = expected_damage
```

Comment:
A message to notify the user that they've been infected with spores or something would be appreciated

Reply:
I really feel this would benefit from using an actual seed item, that embeds in some bodypart. It would make the message of infection very clear and would let players try to rip it out from their limb, before it growns into a venus trap.

Reply:
hmm, that really could be a good idea.